Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,56 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            // Person 1:
+            // Gathering Input:
+            // Name
+            Console.WriteLine("Enter Name: ");
+            string nm1 = Console.ReadLine();
+
+            // Age
+            Console.WriteLine("Enter Age: ");
+            int age1 = int.Parse(Console.ReadLine());
+        
+            // Weight
+            Console.WriteLine("Enter Weight: ");
+            int weight1 = int.Parse(Console.ReadLine());
+
+            // Height
+            Console.WriteLine("Enter Height: ");
+            float height1 = float.Parse(Console.ReadLine());
+
+            // Calculating BMI
+            float bmi1 = weight1 / (height1 * height1) * 10000;
```

Comment:
For example, bmi is a well-known abbreviation and full name (Body-Mass-Index) is not needed in this context. So this is correct.